### PR TITLE
#702 fix show queens restaurants under queens

### PR DESCRIPTION
**Pull request by stanzheng**
## [Ticket](insert_trello_card_link_here)

https://trello.com/c/8cuqL3Ps/702-fix-boc-bocs-location-filtering
### [Description of changes]
fix filtering for bocboc

### [Screenshots or clip of change]

image

### CODE REVIEW

**Review by wilsonj806 — APPROVED** [review]
looks good!

File: src/components/MerchantsPage/MerchantsPage.tsx [hunk 1]
```diff
@@ -76,12 +76,14 @@ const MerchantsPage = (props: Props) => {
           store!.locations.length > 0 && store!.locations[0].city === type
       );
       if (type === 'Floral Park') {
-        const flushing = sellers.filter(
+        const queens = sellers.filter(
```
Comment (jacoblb64) [inline src/components/MerchantsPage/MerchantsPage.tsx:79]: This logic confuses me, why is if (floral park) the top level, and does that need to be in the or statement? In any case I trust that this works 😬 thanks for hopping on this!

**Review by jacoblb64 — APPROVED** [review]

File: src/components/MerchantsPage/MerchantsPage.tsx (hunk 1 above)
Comment (stanzheng) [reply src/components/MerchantsPage/MerchantsPage.tsx:79]: so it seems like floral park is the default as it was the first so it was always classified as "queens"

Comment (jacoblb64) [reply src/components/MerchantsPage/MerchantsPage.tsx:79]: Ahhh makes sense, thanks for the clarification!

**Pull request merged**